research-prototype/ipatestinput1.py

```diff
@@ -0,0 +1,43 @@
+import argparse
```

Comment:
why do we need 2 new input files? can we instead refactor the original one and add a descriptive option to the command line interface to choose which version to use? it's unclear to me what the differences are.